cli/flox/src/commands/init/mod.rs

```diff
@@ -809,4 +817,63 @@ mod tests {
             activate_mode: None,
         });
     }
+
+    /// Verify that format_customization() correctly converts InitCustomization to TOML.
+    #[test]
+    fn test_format_customization() -> Result<()> {
+        // Create a test InitCustomization with various fields populated
+        let customization = InitCustomization {
+            hook_on_activate: Some("echo 'Activating environment'".to_string()),
+            profile_common: Some("export COMMON_VAR=value".to_string()),
+            profile_bash: Some("export BASH_VAR=value".to_string()),
+            profile_fish: Some("set -x FISH_VAR value".to_string()),
+            profile_tcsh: Some("setenv TCSH_VAR value".to_string()),
+            profile_zsh: Some("export ZSH_VAR=value".to_string()),
+            packages: Some(vec![CatalogPackage {
+                id: "test-package".to_string(),
+                pkg_path: "test.package".to_string(),
+                version: Some("1.0.0".to_string()),
+                systems: None,
+            }]),
+            activate_mode: None,
+        };
+
+        let toml_str = format_customization(&customization)?;
+        
+        // Use indoc to create the expected TOML with proper indentation
+        let expected_toml = indoc! {r#"
+            version = 1
+
+            [install]
+            test-package.pkg-path = "test.package"
+            test-package.version = "1.0.0"
+
+            [hook]
+            on-activate = """
+              echo 'Activating environment'
+            """
+
+            [profile]
+            common = """
+              export COMMON_VAR=value
+            """
+            bash = """
+              export BASH_VAR=value
+            """
+            fish = """
+              set -x FISH_VAR value
+            """
+            tcsh = """
+              setenv TCSH_VAR value
+            """
+            zsh = """
+              export ZSH_VAR=value
+            """
+        "#};
+
+        // Compare the generated TOML string with our expected output
+        assert_eq!(toml_str, expected_toml);
+
+        Ok(())
```

Comment:
```suggestion
```

Reply:
The test doesn't ever need to return an error because it just panics if there's a failure. So it's slightly more concise to not return a `Result`